app-text/cherrytree/cherrytree-0.99.0_p20200520.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+
+HASH_COMMIT="4e66c82c108bdfb56983d2c3614d88cc61f5179e" # 20200520
+
+inherit cmake python-any-r1 xdg-utils
+
+DESCRIPTION="A hierarchical note taking application (C++ version)"
+HOMEPAGE="https://www.giuspen.com/cherrytree"
+SRC_URI="https://github.com/giuspen/cherrytree/archive/${HASH_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+# GPL-3 — future/src/ct (CherryTree)
+# LGPL-2.1 — future/src/7za (7zip)
+# MIT — future/src/fmt (libfmt)
+LICENSE="GPL-3 LGPL-2.1 MIT"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="nls test"
```

Comment:
this, and BDEPEND block below: nls flag is nowhere used to tick a build system box - leftover from older version or did you forget something in src_configure?